tilde/config/nvim/init.vim

```diff
@@ -26,17 +26,6 @@ colorscheme solarized
 " === Rainbow ===
 let g:rainbow_active = 1
 
-" ======= Syntastic =======
-" First check JSHint, then JSCS
-let g:syntastic_javascript_checkers = ['eslint']
-
-" Also run a Syntax Check when the file is opened
-let g:syntastic_check_on_open=1
-
-" More cats
-let g:syntastic_error_symbol='🙀'
-let g:syntastic_warning_symbol='😿'
```

Comment:
No worries, they'll be coming back – this is WIP 😄 I replaced `syntastic` with `neomake`. Therefore I need to configure this differently!